src/coreclr/vm/jitinterface.h

```diff
@@ -585,6 +585,13 @@ class CEEInfo : public ICorJitInfo
 #endif
 
 protected:
+    struct FieldAddress
+    {
+        void* address;
+        bool  frozen;
+    };
+    typedef MapSHash<CORINFO_FIELD_HANDLE, FieldAddress> FieldHandleAddressMap;
+    FieldHandleAddressMap*  m_fieldHandleAddressMap;
```

Comment:
```suggestion
```

Reply:
Ah, forgot to push commit that removed this, thanks!